src/Accelerators.jl

```diff
@@ -0,0 +1,47 @@
+# should be included in EnsembleKalmanProcess.jl
+
+export DefaultAccelerator, NesterovAccelerator
+
+
+# Since nothing is a possible input to EnsembleKalmanProcess, we do not need a default accelerator (default is nothing)
+"""
+$(TYPEDEF)
+
+Default accelerator provides no acceleration.
+
+$(TYPEDFIELDS)
+"""
+struct DefaultAccelerator <: Accelerator end # can use types/subtypes to ensure v, ens_prev consistent, and to multiple dispatch if multiple cases.
+
+function DefaultAccelerator()
+    return NesterovAccelerator
+end
+
+"""
+$(TYPEDEF)
+
+Accelerator that adapts Nesterov's momentum method for EKI. 
+Stores a second state variable (denoted v, a "momentum" value) and the previous parameter value (denoted ens_prev).
+
+$(TYPEDFIELDS)
+"""
+struct NesterovAccelerator{FT<:AbstractFloat, VV<:AbstractVector} <: Accelerator  ## extra safety to ensure types of v,ens_prev match?
+    r::FT
+    v
+    ens_historical::VV
+end
+
+function NesterovAccelerator()
+    return NesterovAccelerator
```

Comment:
Because the struct you create above has 3 entries, you need this to return some default version e.g. 
```julia

return NesterovAccelerator(3,Float64[],Float64[])
```
So when the user calls `NesterovAccelerator()`, it will return the version with `r=3`